Resources/Prototypes/DeltaV/Actions/types.yml

```diff
@@ -0,0 +1,13 @@
+﻿- type: entity
+  id: ActionOpenRadioImplant
```

Comment:
if made to reuse storage implant this action should inherit from it and just change the user facing fields

Reply:
you inherit the action
and would make it easier with action ecs if it already inherits BaseAction less work to do